tests/clientcmd/command.go

```diff
@@ -36,20 +36,9 @@ import (
 	"kubevirt.io/kubevirt/tests/flags"
 )
 
-func GetK8sCmdClient() string {
-	// use oc if it exists, otherwise use kubectl
-	if flags.KubeVirtOcPath != "" {
-		return "oc"
-	}
-
-	return "kubectl"
-}
-
 func FailIfNoCmd(cmdName string) {
```

Comment:
I think we can get rid of this helper in another cleanup PR. Calling a nonexistent binary will already fail and not all places that require one of the cmds are using this anyway.